gui/default/syncthing/folder/folderPickerModalView.html

```diff
@@ -0,0 +1,30 @@
+<modal id="folderPicker" status="default" heading="{{'Select Folder' | translate}}" icon="fa fa-folder-open" large="yes"
+       closeable="yes">
```

Comment:
Thanks for all the fixes. This looks great and works as expected for me 🙂.

Just one last suggestion, maybe put all this on the same line? All other modal HTML files have it like that, regardless of the length.

```suggestion
       <modal id="folderPicker" status="default" heading="{{'Select Folder' | translate}}" icon="fa fa-folder-open" large="yes" closeable="yes">
```